jmix-flowui/flowui/src/main/resources/io/jmix/flowui/app/bulk/bulk-edit-view.xml

```diff
@@ -0,0 +1,56 @@
+<!--
+  ~ Copyright 2023 Haulmont.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<view xmlns="http://jmix.io/schema/flowui/view"
+      title="msg://bulkEditor.title">
+    <actions>
+        <action id="applyChanges"
+                text="msg:///actions.Apply"
+                icon="CHECK"
+                actionVariant="PRIMARY"
+                shortcutCombination="${SAVE_SHORTCUT}"/>
+        <action id="viewClose"
+                type="view_close"/>
+    </actions>
+
+    <layout expand="fieldLayoutScroller">
+        <div id="infoDiv"
+             text="msg://bulkEditor.info"
+             themeNames="badge"/>
+
+        <scroller id="fieldLayoutScroller"
+                  scrollBarsDirection="VERTICAL"
+                  width="100%">
+            <formLayout id="fieldLayout">
+                <responsiveSteps>
+                    <responsiveStep minWidth="0em"
```

Comment:
You don't need to define size unit for 0 (even if studio forces you :-))
Also, let's increase `minWidth` for steps, something like: 
1 - 0
2 - 60em (we can't use 64em even though it's a dialog size, because actual form container size is lesser and by default it will has 1 column)
3 - 120em

With current values, field can be too small.